Servers/UI/OJS.Servers.Ui/ClientApp/src/components/common/CollectionKeyUtils.ts

```diff
@@ -0,0 +1,15 @@
+﻿const defaultKeyFunc = <TValue extends unknown>(value: TValue) => {
```

Comment:
Convention for **non-component** files are `snake-case`, i.e. `collection-key-utils`